tests/unit/s2n_ecdsa_test.c

```diff
@@ -173,25 +174,8 @@ int main(int argc, char **argv)
         /* Reset signature size when we compute a new signature */
         signature.size = maximum_signature_length;
 
-        /* Not all hash algorithms are supported for EVP ECDSA signing.
-         * See s2n_evp_signing_validate_hash_alg.
-         */
-        bool hash_is_md5 = (hash_alg == S2N_HASH_MD5 || hash_alg == S2N_HASH_MD5_SHA1);
-        bool hash_is_supported = !(hash_is_md5 && s2n_is_in_fips_mode());
```

Comment:
Notice that this "which hash algs are valid" logic didn't account for SHA1+ECDSA like s2n_evp_signing_validate_hash_alg does. That was fine before because it used a variation of the signing logic that didn't care. I plan to switch to a variation of the signing logic that does care.